### ISSUE TITLE
Thanks and FYI

### ISSUE THREAD

**Issue by ladew222** [tomquisel/covid19-data#15]
I am building a Map/dashboard using your data. Thank you!  I am pulling in some additional information from census which allows it to switch to  population relative Choropleths along with some other information.  It is in it infancy but is using D3Plus. Thought I would let you know. Always willing to share.

**Comment by tomquisel** [comment]
That's wonderful! If you're able to contribute back logic that pulls the population for each county into the CSVs, that'd be a huge win! ❤️

**Comment by ladew222** [comment]
Of course, I will send it along. I pulled the census data and American community survey data from NHGIS, which I put into a csv. I then merge the two into one JS object. I then use this as the basis for my map data—but it could easily be exported out to a larger csv.

—Eric

**Comment by ladew222** [comment]
I have a (semi) working version now online in the depositor https://github.com/ladew222/CVD-Geo <https://github.com/ladew222/CVD-Geo> and the live page is at https://ladew222.github.io/CVD-Geo/ <https://ladew222.github.io/CVD-Geo/>. It uses d3 for the data component.  The function that does the data mergining is title(for better or worse) get_data in the index.html.  The interface is still not perfect, but you can load days and filter by a few variables which more visible on hover. I also have a rough pearsons option to look at correlations—which has not been flushed out fully.  I need to go through and validate some of the functions and the state filtering in not yet functional as well.  Please let me know if you have any questions/suggestions.

**Comment by ladew222** [comment]
As an FYI, I have made significant changes to to the data merge as there were a couple of issues with type conversion and digit length. I probably sent it along too soon.  The data tool has also had updates.

**Comment by tomquisel** [comment]
Thanks @ladew222 is it in a good place now?

**Comment by ladew222** [comment]
It is now. It needed some clean up as I was focused on getting it running more than good. But it is in a good place. I have also brought in data from Google with the help of another user and project that extracted that from charts on pdfs. I put the data into a similar for and uploaded it to my site. It dates back about a month and goes till 3/30.  That data is now mapable on the choropleth. Users can now run a basic Pearson correlation test on whole bunch of variable up against Confirmed at the state or national level. I could allow for more options but I could also be giving the user too many already.  Let me know if you have any requests and or suggestions.  And thanks again for your work. 

https://ladew222.github.io/CVD-Geo/index.html

**Comment by ladew222** [comment]
As an FYI, the live page is now at [balefire.info](http://www.balefire.info) for connivance but is hosted on the same rep
